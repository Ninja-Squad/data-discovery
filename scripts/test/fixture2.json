[
  {
    "name": "123",
    "desc": "I'm a description!"
  },
  {
    "noname": "456",
    "desc": "I'm another description!"
  },
  {
    "name": ["789"],
    "desc": "I'm yet another description!"
  },
  {
    "name": 101112,
    "desc": "I'm still yet another description!"
  }
]