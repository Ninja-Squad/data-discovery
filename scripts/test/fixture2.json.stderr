"WARNING: Following document has been dropped from bulk because it misses the given key identifier (name): {\"noname\":\"456\",\"desc\":\"I'm another description!\"}"